pkgs/data/fonts/fira-math/next.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub, fira-math, python3 }:
+
+fira-math.overrideAttrs (oldAttrs: {
+  pname = oldAttrs.pname + "-next";
+  version = "unstable-2023-10-09";
```

Comment:
Although old, there is a previous version, so according to the [README](https://github.com/NixOS/nixpkgs/tree/81837893610744bdcb42a1b4a8811c50cca63122/pkgs#package-naming) the version scheme should be `0.3.4-unstable-${date}`